lib/features/about_us_view/widgets/contact_section.dart

```diff
@@ -27,17 +28,17 @@ class ContactSection extends StatelessWidget {
   }
 }
 
-class _ContactIcon extends StatelessWidget {
+class _ContactIcon extends ConsumerWidget {
   const _ContactIcon({required this.url, required this.icon});
 
   final String url;
   final String icon;
 
   @override
-  Widget build(BuildContext context) {
+  Widget build(BuildContext context, WidgetRef ref) {
     return GestureDetector(
       onTap: () async {
-        await LaunchUrlUtil.launch(url);
+        await ref.launch(url);
```

Comment:
Can we use here `unawaited`?

Reply:
Sure, but also there's nothing after it, so 
```dart
onTap: () async => ref.launch(url),
```
should have similar effect I believe